monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -336,29 +245,12 @@ def predict(self, predict_params=None):
     "https://github.com/Project-MONAI/research-contributions/releases/download/algo_templates/4e1a20e.tar.gz"
```

Comment:
We need to update the zip package, otherwise the templates it downloads doesn't have the right fill_template_config methods